### DISCUSSION TITLE
AU Core PractitionerRole Consensus Discussion

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#38] [Consensus Discussions for R1]
Check out  [AU Core PractitionerRole](http://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitionerrole.html), noting the draft [general conformance](http://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html) and [Must Support](http://build.fhir.org/ig/hl7au/au-fhir-core/must-support.html) data obligations.

_To see the full constraints inherited from AU Base view the Key Elements Table_

AU Core PractitionerRole should support the minimum requirements to record, search, and fetch basic demographics and other administrative information about an individual practitioner in a role, associated with a patient.

Comment on:
- If this profile was mandated tomorrow, what problems would that cause for your organisation?
- What's missing?
- Should any other identifier besides HPI-I and Medicare Provider Number be supported in AU Core? What about Employee Identifier?
- Should anything change?

**Comment by heathfrankel** [comment]
Please allow active value of false to support cases when clinical resource reference a PractitionerRole that is no longer active within an organization

HPI-I should NOT be provided on PractitionerRole since it is associated with the individual, not on each role or at each location the practitioner operates.

The Employee Identifier would be nice to have given that the My Health Record allows this as an alternative identifier to the HPI-I (or is this no longer allowed).

A mandatory identifier may also be problematic for scenarios when staff do not have a provider number and employee numbers are not used.

**Reply by kimberley-hilton** [reply]
Agreed re HPI-I

**Reply by utsavigajjar** [reply]
Agree re HPI-I

**Reply by dt-r** [reply]
Discussed in [2023-09-28 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=184920587).

- Motion to remove support for HPI-I on PractitionerRole passed. Change tracked via https://github.com/hl7au/au-fhir-core/issues/58.
- Support for Employee identifier (which is supported by My Health Record) be used instead? 
   - Outcome: Agreement that Employee identifier is useful, is allowed but the profile, no interest in explicit must support in the profile
- Motion to remove the mandatatory cardinality on PractitionerRole.identifier passed. Change tracked via https://github.com/hl7au/au-fhir-core/issues/57

_NOTE: Outcome of the two above changes is that no specific type of identifier is supported for this resource type and identifier is not mandatory._

Discussed in [2023-10-19 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=184930419)

- Motion to remove PractitionerRole.active passed. Change tracked via https://github.com/hl7au/au-fhir-core/issues/61

**Comment by dt-r** [comment]
From @ir4y

I have a question about the fixed value for the active field in Patient, Practitioner, PractitonerRole, and Organization.
Could you please share a bit more details on why it is a fixed true value?

There are multiple use cases when you need to make a resource inactive. @heathfrankel mentioned some of them in the GitHub discussion (please check related topics for dedicated resources) and all of them make sense to me.

_Originally posted by @ir4y in https://github.com/hl7au/au-fhir-core/discussions/43_

**Reply by dt-r** [reply]
Discussed in [2023-10-19 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=184930419)

Motion to remove PractitionerRole.active passed. Change tracked via https://github.com/hl7au/au-fhir-core/issues/61

**Comment by dt-r** [comment]
PractitionerRole.healthcareService should be 0..* in AU Core PractitionerRole.

What is being defined by that cardinality is: is it meaningful to limit the set of services provided by a practitioner in a role at a location for an organisation to only one? Effectively saying that we
* treat a practitioner in a role at a location for an organisation as providing a single service and by unique it has it’s own record and identifier that are managed. Thus 0..1.
* treat a practitioner in a role at a location for an organisation as providing more than one service, and these would have a meaningful record and identifier that goes across the set of services. Thus we should allow 0..*

I think this should be 0..*.

**Reply by heathfrankel** [reply]
The meaning and use of healthcareService needs to be clearly determined before we make this decision (although I am not necessarily in disagreement with suggestion). The PD2 IG has used HealthcareService in a manner that is more constrained (a service centre that delivers healthcare services at a single location by an organization) than my interpretation of the resource in the base FHIR specification (support multiple locations), which has resulted in the 0..1 cardinality (because the Medicare Provider Number related to the PractitionerRole should only be relevant to one location) . 
Given the existing use of the PD2 IG within Australia, and its relevance to health information exchange, this needs to be carefully reviewed and how it relates to Location.
I notice it is not on our hot list of resources and hence, resources that reference it should probably be agnostic until this is better understood. 
Perhaps remove it from the profile altogether? 
We already have location, although this should be 0..1 given that the provider number is only relevant to one location.

**Reply by dt-r** [reply]
I'd count HealthcareService as in the hot list. It wasn't raised in the very first TDG meeting when asked to list their top 10, but it's currently number 11 in terms of [up votes to be in R1](https://github.com/hl7au/au-fhir-core/discussions/categories/au-core-r1-resource-priority-list?discussions_q=is%3Aopen+category%3A%22AU+Core+R1+Resource+Priority+List%22+sort%3Atop+created%3A%3E%3D2022-09-26).

**Reply by dt-r** [reply]
I think don't think Provider Directory is the more abstract case. I think that's AU Core. Provider Directory necessarily needs to support some specific constraints to drive that use case.

**Reply by mmalc** [reply]
@heathfrankel Is it worth noting that the Location should be [0..1] if the identifier is [AUMedicareProviderNumber](https://build.fhir.org/ig/hl7au/au-fhir-base//StructureDefinition-au-medicareprovidernumber.html)
If the provider role identifier is HPI-I, I'm assuming the "Id of a practitioner at a particular practice" rule doesn't apply.
Or are we thinking to constrain the use of PractitionerRole to infer the use if a Medicare Provider ID?

**Reply by dt-r** [reply]
I think we are moving to the position that PractitionerRole.location isn't core of the core and we want to be silent on in AU Core. We didn't quite finish that discussion previously and should look at this, maybe in the October 19 meeting.

**Reply by dt-r** [reply]
Discussed in [2023-10-19 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=184930419)

Motion to remove must support and constraints on PractitionerRole.healthcareService. Change tracked via https://github.com/hl7au/au-fhir-core/issues/56

**Comment by heathfrankel** [comment]
Mandatory Search parameter for practitioner by reference I suspect is unuseful, the chained practitioner.identifier would be more likely.
If HPI-I is removed from the PractitionerRole, then it should be removed from the identifier search parameter.
Providing a code only in an identifier search parameter is ambiguous. If allowed, I would suggest stating a single (or explicit list) type of identifier that will be used in the search.
a chained organization.identifier may be better or additional search parameter than the organization reference.
I suggest removing the mandatory requirement to support _revinclude of provenance until we at least have a profile for it.

**Reply by nielmdrec** [reply]
We use PractitionerRole for our Contacts (e.g. Health Professionals) and it's possible that some Contacts don't have identifiers.
This will be an issue because AU Core PractitionerRole requires at least 1 (1..*) identifier.

Also, Contacts can be inactivated in our system, so we need support for PractitionerRole.active = false

**Reply by dt-r** [reply]
If systems are missing supported AU Core data, then the AU Core definition of [Missing Data](http://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html#missing-data) would apply. That's something we need to look at with the context of how elements are interpreted through [Must Support](https://build.fhir.org/ig/hl7au/au-fhir-core/must-support.html) and [Conformance Requirements](https://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html).

**Reply by utsavigajjar** [reply]
Even if a practitioner does not have a national identifier, when we persist their information in our system, we must assign them some form of identifier, can that be used as an identifier (although may not be of much value if its an internal org. specific identifier).

**Reply by dt-r** [reply]
To be progressed via https://github.com/hl7au/au-fhir-core/issues/80 and https://github.com/hl7au/au-fhir-core/issues/70

**Comment by dylan-jf7** [comment]
If Identifier is not made mandatory can we add an additional conformance requirement that at least one of Organisation, Healthcare Service, or Location(maybe) mandatory

**Reply by dt-r** [reply]
Agreed. This already exists for Organization, Location, HealthcareService. For profile of an entity type resource (Patient, Organization, etc.) if identifier isn't mandatory then we need a constraint that says that one of the accepted identifying elements need to be supplied e.g. name or identifier or ...

For PractitionerRole my preference is:
`The practitioner role shall at least have a valid identifier or practitioner or code`

We should discuss this. Maybe in the October 19 zoom call.

**Reply by dt-r** [reply]
To be progress via https://github.com/hl7au/au-fhir-core/issues/79

**Comment by dt-r** [comment]
This discussion is now closed. Feedback on AU Core should be raised using GitHub issues. Feedback might be suggesting a feature, pointing out a place where a specification is unclear or over-restrictive, identifying a spelling or grammar issue, identifying a defect, or suggesting that an entire area needs to be rethought.

Integration with HL7 JIRA is coming and when available specification feedback will be migrated to JIRA. An announcement will be posted in this discussion when it is available.

Questions on this profile can be raised in chat.fhir.org: https://chat.fhir.org/#narrow/stream/179173-australia
